HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+## 1.0.9 / 2021-06-11
+
+- Replace deprecated package Request with Axios
```

Comment:
if we do get rid of security issues from lodash might be good to call this out here, since that is a net positive too

Reply:
Agreed